packages/toast/src/ToastEmitter.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Toast, {
+    ToastProps as ImportedToastProps,
+    ToastConfig
+} from "react-native-toast-message";
+import { CustomToastTranslator } from "./components/CustomToastTranslator";
+import { ToastTypes } from "./types";
+
+const toastConfig: ToastConfig = {
+    [ToastTypes.ERROR]: params => <CustomToastTranslator {...params} />,
```

Comment:
It seems like react-native-toast-message is the culprit behind this type of config. Is there no way around this? Seems redundant the way it is now since all configurations point to the same component

Reply:
Don't think it's a way around it :(

Reply:
I simplified the config a bit now though, take a look